src/test/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -50,6 +50,11 @@ const paymentIntentionInInstallment = {
   }
 }
 
+const paymentIntentionBySetDate = {
+  paymentOption: PaymentOption.BY_SPECIFIED_DATE,
+  paymentDate: MomentFactory.currentDate().add(5, 'days')
```

Comment:
Can you make the by set date paymentDate further in the future? say 3 months? pay immediately has a paymentDate of 5 days and this may confuse things when people are looking at the tests.